ui-modules/utils/server-status/server-status.js

```diff
@@ -65,6 +65,9 @@ export function BrServerStatusDirective() {
                 }else if(response.status === 404) {
                     state = BrServerStatusModalController.STATES.NO_CONNECTION;
                 }else if(response.status === 401 || response.status === 403 ) {
+                    if( response.headers("LOGIN_PAGE")) {
+                        $window.location.href = '/' + response.headers("LOGIN_PAGE");
```

Comment:
As before I'm not sure this is the angularjs way

Reply:
Frankly, I do not know this bit, look at the `$location` service docs https://docs.angularjs.org/guide/$location

Reply:
My vote is to redirect it from the backend.